src/element/binding.ts

```diff
@@ -183,7 +183,9 @@ const bindLinearElement = (
   });
   mutateElement(hoveredElement, {
     boundElementIds: [
-      ...new Set([...(hoveredElement.boundElementIds ?? []), linearElement.id]),
+      ...Array.from(
+        new Set([...(hoveredElement.boundElementIds ?? []), linearElement.id]),
+      ),
     ],
```

Comment:
let's assign the `Array.from` directly to `boundElementIds` to save some CPU (can't suggest commit FSR)